apps/ledger-live-mobile/src/screens/ReceiveFunds/ReceiveSecurityModal/InitMessage.tsx

```diff
@@ -16,12 +19,33 @@ type Props = {
 };
 
 const InitMessage = ({ setStep, onVerifyAddress }: Props) => {
+  const route = useRoute();
+
   const onDontVerify = useCallback(() => {
+    track("button_clicked", {
+      button: "Reveal my address without verifying",
+      screen: route.name,
+      drawer: "initMessage",
+    });
     setStep("confirmUnverified");
-  }, [setStep]);
+  }, [setStep, route]);
 
+  const handleVerifyAddress = useCallback(() => {
+    track("button_clicked", {
+      button: "Verify my address",
+      screen: route.name,
+      drawer: "initMessage",
+    });
+    onVerifyAddress();
+  }, [route, onVerifyAddress]);
   return (
     <Flex flex={1} justifyContent="center" mt={3}>
+      <TrackScreen
+        category="ReceiveFunds"
+        name="Verification Security Disclaimer"
+        source={usePreviousRouteName()}
```

Comment:
just a small feedback here, but please use a variaable to set the result of `usePreviousRouteName()` because it would be very easy to accidentally break the "rule of the hooks" by making some conditional JSX.